lib/spack/spack/util/compression.py

```diff
@@ -186,47 +182,47 @@ def _lzma_decomp(archive_file):
         with open(archive_out, "wb") as ar:
             with lzma.open(archive_file) as lar:
                 shutil.copyfileobj(lar, ar)
+        return archive_out
     else:
-        if sys.platform == "win32":
-            return _7zip(archive_file)
-        else:
-            return _xz(archive_file)
+        return _xz(archive_file)
```

Comment:
(question) You mention:

> Now that we've dropped support for these older version of Python, we can consistently rely on the lzma Python module existing to extract Clingo for installation.

but given that `xz` is now available on Windows, it wouldn't matter if there was an older version of Python, if I understand correctly.

Reply:
`xz`, like `gzip`, and `bz2` is not necessarily available on Windows now, this is just a consistency change. Ideally we're always going to use the python modules.